mappings/net/minecraft/level/chunk/Chunk.mapping

```diff
@@ -19,25 +21,93 @@ CLASS net/minecraft/class_43 net/minecraft/level/chunk/Chunk
 		ARG 2 tiles
 		ARG 3 x
 		ARG 4 z
+	METHOD method_858 equals (II)Z
+		ARG 1 x
+			COMMENT The chunk's x coordinate
+		ARG 2 z
+			COMMENT The chunk's z coordinate
 	METHOD method_859 getTileId (III)I
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD method_860 setTile (IIII)Z
+		ARG 1 x
+			COMMENT The relative X coordinate
+		ARG 2 y
+			COMMENT The relative Y coordinate
+		ARG 3 z
+			COMMENT The relative Z coordinate
+		ARG 4 tileId
 	METHOD method_861 setTileWithMetadata (IIIII)Z
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+		ARG 4 tileId
+		ARG 5 meta
 	METHOD method_862 placeTileEntity (IIILnet/minecraft/class_55;)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 tileEntity
+	METHOD method_863 createRandom (J)Ljava/util/Random;
+		ARG 1 seed
+	METHOD method_864 getLightLevel (Lnet/minecraft/class_56;III)I
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_865 setLightLevel (Lnet/minecraft/class_56;IIII)V
+		ARG 1 type
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 light
+	METHOD method_866 appendEntities (Ljava/lang/Class;Lnet/minecraft/class_25;Ljava/util/List;)V
+		ARG 1 entityClass
+		ARG 2 box
+		ARG 3 entities
+	METHOD method_867 addTileEntity (Lnet/minecraft/class_55;)V
+		ARG 1 tileEntity
 	METHOD method_868 addEntity (Lnet/minecraft/class_57;)V
 	METHOD method_869 removeEntity (Lnet/minecraft/class_57;I)V
+	METHOD method_870 appendEntities (Lnet/minecraft/class_57;Lnet/minecraft/class_25;Ljava/util/List;)V
+		ARG 1 except
+		ARG 2 box
+		ARG 3 entities
+	METHOD method_872 fillChunkData ([BIIIIIII)I
+		ARG 1 data
+		ARG 8 length
 	METHOD method_873 generateHeightmap ()V
 	METHOD method_874 getHeight (II)I
+	METHOD method_875 getMetadata (III)I
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_876 setMetadata (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 metadata
 	METHOD method_877 removeEntity (Lnet/minecraft/class_57;)V
 		ARG 1 entity
 	METHOD method_879 isAboveGround (III)Z
 		ARG 1 localX
 		ARG 2 y
 		ARG 3 localZ
+	METHOD method_880 getAbsoluteLight (IIII)I
+		COMMENT @return the combined sky and block light level of a certain position
```

Comment:
"combined" to me sounds like it's adding them together, which I believe is not what is happening here. Iirc it's more like max(sky, block) but sky has a bit more maths to deal with ambient light for day/night